### DISCUSSION TITLE
Add PaymentStatus Model to OrdersShopaholic Plugin

### DISCUSSION THREAD

**Discussion by igor-tv** [lovata/oc-shopaholic-plugin#391] [Ideas]
Having two separate models (OrderStatus and PaymentStatus) will allow more flexible order management. This should make it easier to implement two-step payments with a hold, as well as refunds for orders from the backend interface.

**Comment by lautsevich** [comment]
Looks reasonable! Thx for the suggestion! @kharanenka, how do you feel about that?

**Reply by kharanenka** [reply]
Hi! I think it is good idea.
